evap/evaluation/templates/contact_modal.html

```diff
@@ -1,4 +1,6 @@
-<div class="modal fade" id="successMessageModal_{{ modal_id }}" tabindex="-1" role="dialog" aria-labelledby="successMessageModalLabel_{{ modal_id }}" aria-hidden="true">
+{% load static %}
+<div class="modal fade" id="successMessageModal_{{ modal_id }}" tabindex="-1" role="dialog"
+     aria-labelledby="successMessageModalLabel_{{ modal_id }}" aria-hidden="true">
```

Comment:
Adding a linebreak is probably a good idea, but I think indenting with 5 spaces for that is rather odd (literally, har har).

@niklasmohrin what's the current formatting rule here?

Reply:
yes, that is quite co-even

in general, indent 4 spaces, everything is else is kind of do as you please. For Python, our lines are max. 120 chars long, but in html we are not as strict. Personally, I always try to do what `prettier` does, which is [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIwA2AhgM4kC8AOiALYTpEG4BmR6c1um6VIJArmDBwyAWREkiAczij6jAPrBguOgwILuuAL7bOMIgCNMUdgA9eAWgCMnAE4QCcXlkYQp1SlFw+fRO5hElsSGcARO6IYAnrwCQhLiZNKy8gQAMkZhSipqilq6nP6BlgAW3OxQvDB2-BwgAHxevrgYOPjEZLy5BJauBO72js7U6BBg-DQIMNSN3s2+rXiEpBTU3ZaQsFMzTfN7i+0rXamlcGxwdjtzezctJQCsh51rJzCYME6c3LGCwmISyTk6gyoQ0ylUqU06B0egawAApLhqkQoCRcAByRKSGS4EhTdG4eHaVAAeges1uzVJWGwFMpPgOy2etBOhnoUSu9L2CKRdhRaPRAE0IPw7KoATiAO6kXG-CTMfjhKK4qYAOgJRN29OpODp+xJNL1LQNuq8OtpIAANCAIAAHN7QEjIUD+BySgAK-gQTpQjGlUSd1sMfLAAGs4DAAMpESZpExwZCsAh4oMh8NR21EMAmKTIaq1a1wGihdDsdAZKBSfjJABiEDsNCIMDeleQICI-BgECtIBKMBoBAA6mV4CRM8JI973jh3lE22AyD2THi7DB3XypI3E4wUyAAFYkMyRnNOACK-Ag8G3ybg1szdhXbb7A57toCsEH3BgJWQ1gATAADHeDh4oOfK2m2b4iBc2AJtaACOF7wOudo+u2JCWFAcBwGWPZ2HAiGYPh67SFuSBJrueI0JgeY1LefAnnA56Xgm5E7vRBiGJ+6DfsgAAs1rIpgBA5gAwhANBkSAIj3D2-B4gAKkYPoUfR2C1AAkqYUyRmAAT2gAgqYkYwFETjXniuhAA)

Reply:
Apart from [djhtml](https://github.com/rtts/djhtml) (indenter), I found [djlint](https://github.com/Riverside-Healthcare/djLint) (formatter and linter) which seems worth looking into. We might analyze if we can/cannot configure it to your needs. By default, it is a bit different than `prettier`:
```
<div class="modal fade"
     id="successMessageModal_{{ modal_id }}"
     tabindex="-1"
     role="dialog"
     aria-labelledby="successMessageModalLabel_{{ modal_id }}"
     aria-hidden="true">
    <div class="modal-dialog" role="document">
        <div class="modal-content">
            <div class="modal-header">
                <h5 class="modal-title" id="successMessageModalLabel_{{ modal_id }}">{% trans 'Message sent' %}</h5>
            </div>
            <div class="modal-body">{% trans 'Your message was successfully sent.' %}</div>
        </div>
    </div>
</div>
```
In the meanwhile, I just remove the introduced formatting changes

Reply:
Made #1777 for the discussion. Resolving this for now.